src/python/pants_release/generate_github_workflows.py

```diff
@@ -1028,19 +1028,19 @@ def release_jobs_and_inputs() -> tuple[Jobs, dict[str, Any]]:
     pypi_release_dir = "dest/pypi_release"
     helper = Helper(Platform.LINUX_X86_64)
     wheels_jobs = build_wheels_jobs(
-        needs=["determine_ref"], for_deploy_ref=gha_expr("needs.determine_ref.outputs.build-ref")
+        needs=["release_info"], for_deploy_ref=gha_expr("needs.release_info.outputs.build-ref")
     )
     wheels_job_names = tuple(wheels_jobs.keys())
     jobs = {
-        "determine_ref": {
+        "release_info": {
             "name": "Determine the ref to build",
```

Comment:
Could probably adjust the name of this Job to mention that it is also maybe making the release?

Alternatively, put the "make the release" part in a separate Job rather than in a Step.